presto-main/src/main/java/com/facebook/presto/execution/FragmentResultCacheContext.java

```diff
@@ -85,4 +106,9 @@ public CanonicalPlanFragment getCanonicalPlanFragment()
     {
         return canonicalPlanFragment;
     }
```

Comment:
remove this method?

Reply:
For the scope of this PR, we could remove `canonicalPlanFragment` and its getter method in this class. We probably would introduce them back when trying to support partition-stats based predicate stripping.